src/models/msg/types.ts

```diff
@@ -1 +1 @@
-export type Categories = 'bank' | 'crisis' | 'distribution' | 'governance' | 'slashing' | 'staking' | 'profiles' | 'others';
+export type Categories = 'bank' | 'crisis' | 'distribution' | 'governance' | 'slashing' | 'staking' | 'profiles' | 'client' | 'channel' | 'connection' | 'transfer' | 'others';
```

Comment:
```
export type Categories = 'bank' | 'crisis' | 'distribution' | 'governance' | 'slashing' | 'staking' | 'profiles' | 'ibc' | 'ibc-transfer' | 'others'
```

Reply:
The message categories have been updated